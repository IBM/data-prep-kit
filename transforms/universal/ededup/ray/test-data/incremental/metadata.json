{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "ededup",
    "job type": "ray",
    "job id": "job_id",
    "start_time": "2024-08-20 21:54:19",
    "end_time": "2024-08-20 21:54:20",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "doc_column": "contents",
    "doc_id_column": "document_id",
    "use_snapshot": true,
    "snapshot_directory": "/Users/borisl/Projects/data-prep-kit/transforms/universal/ededup/ray/test-data/input/snapshot",
    "hash_cpu": 0.5,
    "num_hashes": 2,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".parquet"
    ],
    "number of workers": 2,
    "worker options": {
      "num_cpus": 0.5,
      "max_restarts": -1
    },
    "actor creation delay": 0
  },
  "execution_stats": {
    "cpus": 12,
    "gpus": 0,
    "memory": 14.090177917852998,
    "object_store": 2.0,
    "execution time, min": 0.013
  },
  "job_output_stats": {
    "number of hashes": 3,
    "hash memory, GB": 7.180497050285339e-07,
    "de duplication %": 100.0,
    "source_files": 1,
    "source_size": 36132,
    "result_files": 1,
    "result_size": 19280,
    "processing_time": 0.026,
    "source_documents": 5,
    "result_documents": 0,
    "source_doc_count": 5,
    "result_doc_count": 0
  },
  "source": {
    "name": "/Users/borisl/Projects/data-prep-kit/transforms/universal/ededup/ray/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/borisl/Projects/data-prep-kit/transforms/universal/ededup/ray/output",
    "type": "path"
  }
}