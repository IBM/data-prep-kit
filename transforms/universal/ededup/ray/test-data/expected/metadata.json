{
    "pipeline": "pipeline_id",
    "job details": {
        "job category": "preprocessing",
        "job name": "ededup",
        "job type": "ray",
        "job id": "job_id",
        "start_time": "2024-08-15 14:10:22",
        "end_time": "2024-08-15 14:10:23",
        "status": "success"
    },
    "code": {
        "github": "github",
        "commit_hash": "12345",
        "path": "path"
    },
    "job_input_params": {
        "doc_column": "contents",
        "doc_id_column": "document_id",
        "use_snapshot": false,
        "snapshot_directory": null,
        "hash_cpu": 0.5,
        "num_hashes": 2,
        "checkpointing": false,
        "max_files": -1,
        "random_samples": -1,
        "files_to_use": [".parquet"],
        "number of workers": 3,
        "worker options": {
            "num_cpus": 0.8,
            "max_restarts": -1
        },
        "actor creation delay": 0
    },
    "execution_stats": {
        "cpus": 12,
        "gpus": 0,
        "memory": 15.131352234631777,
        "object_store": 2.0,
        "execution time, min": 0.008
    },
    "job_output_stats": {
        "number of hashes": 3,
        "hash memory, GB": 7.180497050285339e-7,
        "de duplication %": 40.0,
        "source_files": 1,
        "source_size": 36132,
        "result_files": 1,
        "result_size": 36288,
        "processing_time": 0.027,
        "source_documents": 5,
        "result_documents": 3,
        "source_doc_count": 5,
        "result_doc_count": 3
    },
    "source": {
        "name": "/Users/borisl/IdeaProjects/data-prep-kit/transforms/universal/ededup/ray/test-data/input",
        "type": "path"
    },
    "target": {
        "name": "/Users/borisl/IdeaProjects/data-prep-kit/transforms/universal/ededup/ray/output",
        "type": "path"
    }
}
